packages/ui-components/src/components/layout/Table.vue

```diff
@@ -21,7 +20,17 @@
         class="divide-y divide-outline-3 h-full overflow-visible"
         :class="{ 'pb-32': overflowCells }"
       >
-        <template v-if="items.length">
+        <div
+          v-if="loading || !items"
```

Comment:
Undefined `items` now means - show loading bar (for easy feeding in of API data)